src/__tests__/assets/v1/v1.3.0/create_call.json

```diff
@@ -0,0 +1,323 @@
+{
+  "defaultAddress": "0x7cbB62EaA69F79e6873cD1ecB2392971036cFAa4",
+  "released": true,
+  "contractName": "CreateCall",
+  "version": "1.3.0",
+  "codeHash": "0x8155d988823a4f6f1bcbc76a64af8e510c4ce68819290d43cf24956bd24dee82",
+  "addresses": {
+    "canonical": "0x7cbB62EaA69F79e6873cD1ecB2392971036cFAa4",
+    "eip155": "0xB19D6FFc2182150F8Eb585b79D4ABcd7C5640A9d",
+    "zksync": "0xcB8e5E438c5c2b45FbE17B02Ca9aF91509a8ad56"
+  },
+  "networkAddresses": {
+    "1": "0x7cbB62EaA69F79e6873cD1ecB2392971036cFAa4",
```

Comment:
Maybe a smaller subset makes more sense for tests (as discussed in stand-up).

Reply:
Did we discuss that? 🙈 

Those files are 1:1 copies (with only an addition of the `addresses` mapping) of the pre-upgrade JSON assets. Using production assets gives me more confidence in the correctness of the package. I also think crafting the assets requires more effort.

Reply:
That shouldn't be the case, I fixed the tests to check that the old files are a subset of the new files.

Docs on the jest matcher I used: https://jestjs.io/docs/expect#tomatchobjectobject